features/machine/alerting.feature

```diff
@@ -62,3 +62,57 @@ Feature: Alerting for machine-api
       | cluster-autoscaler-operator | openshift-machine-api              | ClusterAutoscalerOperatorDown | # @case_id OCP-26250
       | machine-api-operator        | openshift-machine-api              | MachineAPIOperatorDown        | # @case_id OCP-26248
       | machine-approver            | openshift-cluster-machine-approver | ClusterMachineApproverDown    | # @case_id OCP-26110
+
+  # @author jhou@redhat.com
+  # case_id OCP-25628
+  @admin
+  @destructive
+  Scenario: Alert should be fired when machine is in 'Deleting' state for too long
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I pick a random machineset to scale
+
+    Given I create the 'Ready' unhealthyCondition
+    And I run the :annotate client command with:
+      | n            | openshift-machine-api                       |
+      | resource     | machine                                     |
+      | resourcename | <%= machine.name %>                         |
+      | overwrite    | true                                        |
+      | keyval       | machine.openshift.io/exclude-node-draining- |
+    Then the step should succeed
+    And I register clean-up steps:
+    """
+    When I run the :annotate client command with:
+      | n            | openshift-machine-api                       |
+      | resource     | machine                                     |
+      | resourcename | <%= machine.name %>                         |
+      | overwrite    | true                                        |
+      | keyval       | machine.openshift.io/exclude-node-draining= |
+    Then the step should succeed
+    """
+
+    Given I run the :delete admin command with:
```

Comment:
@jhou1 perhaps we can use `admin ensure` step instead here.

Reply:
@pruan-rht this step is to create the `machine is in 'Deleting' state for too long` scenario. The deletion won't be successful until we add the annotation in line#85 in tear down. Then the `admin ensure` is in the last line of this test could delete the machine to ensure a consistent test env.